pkg/minikube/registry/drvs/virtualbox/virtualbox.go

```diff
@@ -52,6 +52,10 @@ func init() {
 }
 
 func configure(cc config.ClusterConfig, n config.Node) (interface{}, error) {
+	ip, network, err := net.ParseCIDR(hostOnlyCIDR)
```

Comment:
do u mind pasting the out of this PR before and after this PR ?

I believe maybe this one you meant cc.HostOnlyCIDR ?
since hostOnlyCIDR is just the string for the flag name ?